src/main/java/io/fabric8/maven/docker/service/BuildService.java

```diff
@@ -66,7 +66,7 @@ public void buildImage(ImageConfiguration imageConfig, MojoParameters params, bo
                              dockerArchive,
                              getDockerfileName(buildConfig),
                              cleanupMode.isRemove(),
-                             noCache, mergedBuildMap);
+                             noCache, mergedBuildMap, buildConfig.getBuildOptions());
```

Comment:
The argument lists becomes quite lengthy now, so that I have refactored it to use a `BuildOptions` argument object. Makes it much nicer, too to merge the various options into a single place.